agent/app/agent_capability.go

```diff
@@ -498,6 +503,24 @@ func (agent *ecsAgent) appendServiceConnectCapabilities(capabilities []*ecs.Attr
 	return capabilities
 }
 
+func (agent *ecsAgent) appendGuardDutyCapabilities(capabilities []*ecs.Attribute) []*ecs.Attribute {
+	isEnabled, err := pathExists(guardDutyServiceFilePath, false)
+	if err != nil {
+		logger.Warn(fmt.Sprintf("Skipping enabling GuardDutyCapabilities. Error: %v", err))
+	}
+
+	supported, err := isGuardDutySupported()
+	if err != nil || !supported {
+		logger.Warn(fmt.Sprintf("Skipping enabling GuardDutyCapabilities. "+
```

Comment:
maybe make this a debug statement? otherwise we will log a warning every time the agent starts up on windows

Reply:
Changed this to a debug statement.